.github/workflows/test build.yml

```
@@ -0,0 +1,25 @@
+name: Test Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies and build
+        working-directory: ./website
+        run: |
+          npm
+          npm build 
```

Change this to, as stated in [docs](https://github.com/moja-global/community-website/tree/main/website#build): 

```suggestion
          npm run build 
```